site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -0,0 +1,62 @@
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-before.png" />
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = ????
+-->
+<i>How jagged</i>
+</p>
+
+Fortunately (and to much rejoice) wwylele stepped into the ring to wrestle with this issue. They prepared [test programs](https://github.com/wwylele/ctrhwtest/tree/master/texture-copy-test) to help gain an understanding of how hardware handles a TextureCopy in comparison to Citra and implemented an accurately functioning TextureCopy mode for Citra's emulation of the 3DS GPU.
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-after.png" />
+<br />
+<!--
+title_id = 0004000000174600
+commit_hash = 686fde7e526e024716baa3aa3ba887d1a2479d41
+-->
+Now we get an accurate portrayal of deoxys having a hard time
+</p>
+
+
+## [Citra-QT: UI Themes](https://github.com/citra-emu/citra/pull/2804) by [Kloen](https://github.com/kloen)
+
+Kloen has put the time and work into Citra's QT frontent to make it themable! Now users can rejoice in the sweet sweet glory of a dark mode alternate and custom coloring schemes!
+
+<p style="text-align: center; font-size: small; padding: 1%">
+<img style="padding: 0% 0% 1% 0%" height="75%" width="75%" alt="Comparison of Dark Theme and Light Theme" src="/images/entry/citra-progress-report-2017-july/theme-comparison.png" />
+<br />
+<!--
+commit_hash = 686fde7e526e024716baa3aa3ba887d1a2479d41
+-->
+CHOOSE YOUR CHARACTER
+</p>
+
+## [Load Shared Font From System Archive](https://github.com/citra-emu/citra/pull/2784) by [wwylele](https://github.com/wwylele)
+
+**This does not elimate the need for dumping a shared font from a legitimate system.**
+
+This in itself isn't an extremely visible or perceivable change from a user perspective, but it is something that helps us take another stride towards accurately recreating the way the 3DS actually operates. All system data is now uniformly stored in the system archive now that the system font can be included with it. Although, Citra does still fall back to the now deprecated shared_font.bin file that was being dumped before for compatibility's sake. 3DSUtil has been updated to be able to dump everything as a system archive so that Citra may be able to work with it in a way that's more accurate to the actual hardware!
```

Comment:
Change `3DSUtil` to `3dsutils` since [GitHub repo](https://github.com/citra-emu/3dsutils) is called that.

Stylistic nitpick; make file names, repo names, and function/class/namespace names `monospaced` by wrapping them in backticks (`` not '').